roles/matrix-jitsi/defaults/main.yml

```diff
@@ -41,16 +44,17 @@ matrix_jitsi_xmpp_guest_domain: guest.meet.jitsi
 matrix_jitsi_xmpp_muc_domain: muc.meet.jitsi
 matrix_jitsi_xmpp_internal_muc_domain: internal-muc.meet.jitsi
 matrix_jitsi_xmpp_modules: ''
+matrix_jitsi_xmpp_muc_modules: ''
+matrix_jitsi_xmpp_muc_modules_internal: ''
 
 matrix_jitsi_recorder_domain: recorder.meet.jitsi
-
+matrix_jitsi_jibri_recorder_user: recorder
+matrix_jitsi_jibri_recorder_password: ''
```

Comment:
I relocated them next to the "recorder domain" because, while these are Jibri settings, they seem to apply more specifically to the recording feature.

Reply:
These variables have been moved back.